### ISSUE TITLE
Tabs coloring and tooltips info and some UI improvments

### ISSUE THREAD

**Issue by macsunmood** [pbek/QOwnNotes#2573]
#### Expected behaviour

- Add notes tabs coloring with accordance to notes' existing tag colors;
- Show some meaningful information in tabs' tooltips (attached tags, date created/modified, etc.) instead of "Double-click to stick note to tab";
- Would be cool to have tab right-click actions menu manually configurable!
- UI/UX helper - highlight notes in the notes list on tabs mouse-hover (different color than the focused note) and vice-versa. This could be an optional trigger.

**Comment by pbek** [comment]
Thank you for your suggestions.

> Add notes tabs coloring with accordance to notes' existing tag colors;

I don't think a QTabWidget can do that.

> UI/UX helper - highlight notes in the notes list on tabs mouse-hover (different color than the focused note) and vice-versa. This could be an optional trigger.

I don't think a QTabWidget has a signal for hovering.

**Comment by macsunmood** [comment]
As metioned at https://www.qtcentre.org/threads/2986-How-to-change-color-of-Tab-in-QTabWidget : 

> You will need to subclass QTabWidget and QTabBar. In the reimplemented version of QTabBar you can changed the color of any tab.
> Use the protected function - setTabBar() of QTabWidget to set your custom tab.

**Comment by pbek** [comment]
Yeah, I guess it would be possible to create your own QTabBar for that to work. Pull requests welcome. 😉

**Comment by macsunmood** [comment]
> Yeah, I guess it would be possible to create your own QTabBar for that to work. Pull requests welcome. 😉 

Okay, now I getting closer to consider getting in with c++ , so I'll be able to make the qownnotes pull requests  .
